Proposals/state-of-the-community.md

```diff
@@ -0,0 +1,36 @@
+# State of The Community
+
+* Proposal: [PR#n]([link to PR])
+* Authors: [Matt Dias](@mdiasdev)
+* Review Manager: [Name]([github profile])
+* Status: [Proposed]()
+
+## Introduction
+
+There has been an interesting shift in the community. Where people used to gather around iOS or macOS as topics, everyone seems to have instead moved to all gather around Swift. I think it would be interesting to hear from a group of community builders and members about the state of the community, what it means to them, and how others can help keep up the spirit of building together.
+
+## Motivation
+
+To discuss the state of the community, how it has changed, how it should change, and help others get involved.
+
+## Proposed contribution
+
+I can represent a community leader (I help organize a Meetup). Otherwise, I'm happy to help reach out to potential hosts.
+
+## Source compatibility
+
+This podcast is for the community by the community. As such, I think it's important to regularly take a health check and reevaluate the state of things to seek out areas of improvement -- a retro of sorts.
+
+## Alternatives considered
+
+I think the existence of the podcast is a nod to the state of the community (in that is thriving). However, I don't think I've heard representatives from the community discuss the community itself in an attempt to motivate others to get involved.
+
+## Roles
+TBD
+
+- Lead Host:
+- Co-hosts:
+- Editor:
+- Publisher:
```

Comment:
I can publish the episode. In the future this might be more open/automated but for now its via Squarespace.